app/assets/javascripts/components/menu_plan_dropdown.js.jsx.coffee

```diff
@@ -0,0 +1,22 @@
+class @MenuPlanDropdown extends React.Component
+  @propTypes =
+    recipes: React.PropTypes.object
+    deleteRecipe: React.PropTypes.func
+
+  removeRecipe: (recipe) =>
+    @props.deleteRecipe(recipe)
+    @setState({recipes: @props.recipes })
```

Comment:
😂 I'll change that. Figuring out how to write `JSX` in coffee had me all screwed up.